CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Next Release
```

Comment:
It's a place holder for contributors creating new PR's. I've just added a note to CONTRIBUTING.md to make note of that.